### PR TITLE
Add Mapillary AB

### PR DESCRIPTION
**Pull request by m-rey**

### CODE REVIEW

File: companies/mapillary-com.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            "desc": "Address",
+            "type": "address"
+        },
+        {
+            "desc": "Username",
+            "type": "input"
+        }
+    ],
+    "request-language": "en",
+    "suggested-transport-medium": "email"
```
Comment (baltpeter) [inline companies/mapillary-com.json]: With a `support@` email, we can unfortunately not suggest email as the transport medium.

File: companies/mapillary-com.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "relevant-countries": [
```
Comment (baltpeter) [inline companies/mapillary-com.json]: Please add the slug.

File: companies/mapillary-com.json (hunk 1 above)
Comment (m-rey) [reply companies/mapillary-com.json]: [Their website](https://www.mapillary.com/privacy) states 

> You can always contact Mapillary at support@mapillary.com if you have any questions regarding your personal data processed under joint controllership, or if you wish to exercise any of your rights under this Policy or the GDPR. 

Should I remove the suggested transport medium in these cases?

Comment (baltpeter) [reply companies/mapillary-com.json]: I agree that this is a little debatable. But our policy is to only suggest email for addresses that clearly *only* go to the privacy team/data protection officer (or at least legal team). If there is a general support mail that is also read by the DPO, we just don't suggest a TM at all.

**Pull request merged**